roles/elastic-stack/ansible-kibana/tasks/build_wazuh_plugin.yml

```diff
@@ -63,9 +63,7 @@
       chdir: "/tmp/app/build"
 
   - name: Install Wazuh Plugin (can take a while)
-    shell: "/usr/share/kibana/bin/kibana-plugin install file:///tmp/app/build/{{ wazuhapp_package_name.stdout }}"
-    environment:
-      NODE_OPTIONS: "--max-old-space-size=3072"
+    shell: NODE_OPTIONS=" {{ node_options }} " /usr/share/kibana/bin/kibana-plugin install file:///tmp/app/build/{{ wazuhapp_package_name.stdout }}
```

Comment:
No need for the extra spaces, you can use: `NODE_OPTIONS="{{ node_options }}"`

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded